real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java

```diff
@@ -85,16 +86,18 @@ public LoggingNode(NodeChannel channel, String outputDirPath, NodeChannel...filt
         statusPub = new Publisher(NodeChannel.NODE_STATUS.getMsgPath());
 
         setupSubscriberList();
-
-        if (!RobobuggyConfigFile.isDataPlayBack()) {
-            setupLoggingTrigger();
-        }
+        setupLoggingTrigger();
 
         statusPub.publish(new NodeStatusMessage(LoggingNode.class, LoggingNodeStatus.INITIALIZED, null));
 
     }
 
-
+    /**
+     * @return the current output file
+     */
+    File getOutputFile() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2376696311/issues/source?bid=2938665&fileBranchId=3199438#l98)